### DISCUSSION TITLE
Use of Random

### DISCUSSION THREAD

**Discussion by PaulLBos** [pybricks/support#668] [Show and tell]
I was struggling to make random work. Eventually came up with this Hope I helps someone else 

```python
#Random number use example

import urandom

#Return the next random floating point number in the range [0.0, 1.0).
value = urandom.random()
print (value)
value = value * 100
print (value)
value = int (value)
print (value)

#Returns a randomly selected element from range(start, stop, step).
value = urandom.randrange(0,100,1)
print (value)
```

**Comment by dlech** [comment]
```python
value = urandom.randrange(0,100,1)
```

FYI, this can be written more simply as:

```python
value = urandom.randrange(100)
```

since 0 is the default start value and 1 is the default step.

We should probably add some more examples and explanation in the docs: https://docs.pybricks.com/en/stable/micropython/urandom.html

**Comment by JobadSayas** [comment]
This is not working in move hub

**Reply by dlech** [reply]
I see you already found https://github.com/pybricks/support/discussions/319. I thought this was mentioned in the docs at one time but I don't see it there now.
